TimedArithmeticQuiz/src/components/QuizComplete.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types"
+import { ResponseList } from "./ResponseList"
+import { QuizHeader } from "./QuizHeader"
+
+export const QuizComplete = ({correctAnswerQuestions, wrongAnswerQuestions, score}) => {
+  return <div>
+    <QuizHeader score={score} />
+    <ResponseList correctAnswers={true} list={correctAnswerQuestions} />
```

Comment:
Rather than passing explicit true and false, we can just specify the parameter on the component

ex - 
```<ResponseList correctAnswers list={correctAnswerQuestions} />```
and
```<ResponseList list={correctAnswerQuestions} />```

Reply:
done